tests/realtikvtest/txntest/txn_state_test.go

```diff
@@ -167,9 +167,16 @@ func TestRunning(t *testing.T) {
 		tk.MustExec("commit;")
 		ch <- struct{}{}
 	}()
-	time.Sleep(100 * time.Millisecond)
-	info := tk.Session().TxnInfo()
-	require.Equal(t, txninfo.TxnRunning, info.State)
+	var state txninfo.TxnRunningState
+	for i := 0; i < 1000; i++ {
+		time.Sleep(time.Microsecond)
```

Comment:
This waiting time is too small, for example, it can be changed to （10*time.Microsecond）